docs/index.md

```diff
@@ -21,12 +21,24 @@ libraryDependencies += "pink.cozydev" %%% "lucille" % "@VERSION@"
 
 ### Parsing
 
-Lucille offers a `parse` function to parse a whole string into a Lucille `MultiQuery` structure:
+Lucille offers a `parse` function to parse a whole string into a Lucille `Query` structure:
 
 ```scala mdoc
 import pink.cozydev.lucille.QueryParser
 
-QueryParser.parse("cats OR dogs")
+QueryParser.default.parse("cats OR dogs")
+```
+
+The default `QueryParser` automatically inserts an `OR` operation inbetween consecutive terms.
+
+```scala mdoc
+QueryParser.withDefaultOperatorOR.parse("cats dogs")
```

Comment:
Perhaps worth noting that `QueryParser.parse("cats OR dogs")` also is OR by default?

Reply:
maybe..... I don't know yet if we should keep the `QueryParser.parse` convenience forever...
So maybe, for now we encourage folks to be explicit with `QueryParser.withDefaultOperatorOR.parse` or `QueryParser.default.parse` and then eventually require it?

For now I think I prefer recommending the explicit thing.